src/losses/ocr.py

```diff
@@ -0,0 +1,127 @@
+import collections
+
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+
+from pathlib import Path
+from src.models import ocr
+from PIL import Image
+from src.disk import disk
+
+class resizeNormalize(object):
```

Comment:
We don't need to worry about compatability with python2, so I don't think we need to inherit from `object` here.

Reply:
It was copypasted, not my code